ansible/jenkins-slave.yaml

```diff
@@ -33,3 +33,7 @@
       prompt: "Enter GitHub user's private SSH key file path"
       default: "~/.ssh/open-power-host-os-builds-bot_id_rsa"
       private: no
+    - name: "known_hosts_file_path"
+      prompt: "Enter upload server's host key file path"
```

Comment:
I'd be more descriptive in this prompt string: "Enter path to file containing known keys for GitHub and upload server hosts"